mars_troughs/custom_acc_models.py

```diff
@@ -63,12 +65,14 @@ def get_accumulation_at_t(self, time: np.ndarray) -> np.ndarray:
         """
         return self.eval(self._var_data_spline(time))
 
+
     def get_xt(
         self,
         time: np.ndarray,
-        int_retreat_model_t_spline: np.ndarray,
+        retr: np.ndarray, # previously known as int_retreat_model_t_spline
```

Comment:
This type of comment is best to put in the docstring for the class.

On line 38 we start the `Args` description. It's out of date, because I think lots of arguments didn't get added to the docstring when they got implemented, but in this case if you document the `retr` argument you would include this comment.